### PR TITLE
fallback to legacy pmml implementation under osgi

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

**Comment by mariofusco** [comment]
jenkins execute fdb

File: kie-internal/pom.xml [hunk 1]
```diff
@@ -87,6 +87,7 @@
               org.kie.api.logger,
               org.kie.api.marshalling,
               org.kie.api.persistence.jpa,
+              org.kie.api.pmml,
```
Comment (gitgabrio) [inline kie-internal/pom.xml:90]: @mariofusco 
Out of curiosity, why are those modifications needed in the Import-Package ?

**Review by gitgabrio — APPROVED** [review]
@mariofusco 
Since this is meant to be a really transitory solution, looks good to me.

**Comment by mariofusco** [comment]
The remaining failures are not OSGi related (which was the intent of this PR) and are currently under investigation.

**Pull request merged**